docs/source/reference/datasets.rst

```diff
@@ -18,6 +18,10 @@ SiameseDataset
 ~~~~~~~~~~~~~~
 .. autoclass:: SiameseDataset
 
+MixUpSoftLabelDataset
```

Comment:
Can you move this before `SiameseDataset`?
We are ordering docs in alphabetical order.